Update Act.json

mira-gruenwald opened the pull request:
I added "act_status" to the Act.json

I hope it works this way ;)

mira-gruenwald committed:
Update Act.json

duncdrum reviewed (approved):
well done this works

mira-gruenwald committed:
Update Person.json

added "narrative_age"

mira-gruenwald committed:
Update readingdata/Person.json

Co-Authored-By: Duncan Paterson <duncdrum@users.noreply.github.com>

mira-gruenwald committed:
Merge branch 'schema-narrative_age' into add-act_status-to-Act.csv

duncdrum committed:
add missing character_status

mira-gruenwald committed:
Update Person.json

duncdrum committed:
add literary_status to PS

mira-gruenwald committed:
Merge branch 'add-act_status-to-Act.csv' of https://github.com/readchina/csv-schema into add-act_status-to-Act.csv

duncdrum committed:
add literary_status to SS

mira-gruenwald committed:
Merge branch 'add-act_status-to-Act.csv' of https://github.com/readchina/csv-schema into add-act_status-to-Act.csv

duncdrum committed:
adjust regex for determining `fictionality`

mira-gruenwald committed:
main_narrator and narrative_time

mira-gruenwald committed:
NarrativePosition

added the schema for NarrativePosition.csv

updated Readme close #4

mira-gruenwald committed:
updated Person.json

mira-gruenwald committed:
Update SecondarySource.json

duncdrum committed:
fix malformed act.json

see readchina/ReadingData#381

mira-gruenwald committed:
adjustments

changed the schema for narrative_time, narrative_status and main narrator.

added a column for the external link of the narrative positions
